### ISSUE TITLE
Include non-referenced assemblies

### ISSUE THREAD

**Issue by jherby2k** [tonerdo/coverlet#192]
My library uses MEF to load "extensions" that are located in a separate directory. I'd like these assemblies instrumented as well. Is there a way to instrument assemblies that aren't project references?

**Comment by jherby2k** [comment]
Taking a look at the code, it seems to just instrument assemblies in the same directory as the test project, minus exclusions. Would you be open to a pull request that allows one to Include additional assemblies? I’m going to start working on it regardless.

**Comment by tonerdo** [comment]
@jherby2k, I'd be open to that contribution. Thanks!

**Comment by tswiftma** [comment]
Yes I need the MEF capability also! I'm currently using OpenCover but need a solution that runs on linux like Coverlet. I would be happy to help test/compare coverage results.

**Comment by jherby2k** [comment]
@tswiftma I have a nuget package for testing this feature at https://www.myget.org/feed/audioworks/package/nuget/coverlet.msbuild, if you want to try it. Do you keep all your MEF components in a single directory, or in subdirectories?

Use either /p:IncludeDirectories="<extension_path>" or add a "\*" to the end if you want to search subdirectories.

**Comment by tswiftma** [comment]
@jherby2k, @tonerdo,  sorry for the slow response I was flat out at work this week. I had to go back to the code and check, we actually use dependency injection. But your change allowed coverlet to see all my dependencies. I ran coverlet vs open cover (both with the same exclusions) with ReportGenerator and the results were virtually identical:

![coverlet](https://user-images.githubusercontent.com/11710469/47228717-e460c300-d393-11e8-9a62-2f36f67c1137.png)

I will dig into the small differences a bit more but this is a big win that I can run coverage directly with dotnet test on linux!
